axelor-account/src/main/java/com/axelor/apps/account/service/moveline/MoveLineTaxServiceImpl.java

```diff
@@ -253,6 +259,14 @@ public int compare(MoveLine o1, MoveLine o2) {
       }
     }
 
+    if ("".equals(oldTaxDescription)) {
```

Comment:
You think it's better to use Strings.isNullOrEmpty(oldTaxDescription)

Reply:
Fixed